### ISSUE TITLE
FileDate

### ISSUE THREAD

**Issue by balzer82** [RobTranquillo/SDM-stadtrat_dresden_metascrape#2]
Das Download-Date ist ja relativ uninteressant. Interessanter als Metainfo wäre ja, wann die PDF auf den Server geladen bzw. gespeichert/erstellt wurde.

**Comment by RobTranquillo** [comment]
Ja, sehe ich auch so. Kommt auch als nächstes mit rein. Es steht auch schon als content im meta-tag "date" in der new.html.
